File: Iniciante/URI 1041.go
package main
 
import (
    "fmt"
)
 
func main() {
    var x, y float64

    fmt.Scanf("%f", &x)
    fmt.Scanf("%f", &y)

    if (x == 0 && y == 0) {
        fmt.Println("Origem")
    } else if (x == 0) {
        fmt.Println("Eixo Y")
    } else if (y == 0) {
        fmt.Println("Eixo X")
    } else if (y > 0 && x > 0) {
        fmt.Println("Q1")
    } else if (y > 0 && x < 0) {
        fmt.Println("Q2")
    } else if (y < 0 && x < 0) {
        fmt.Println("Q3")
    } else if (y < 0 && x > 0) {
        fmt.Println("Q4")
    }
}

File: Iniciante/URI 1041.ml
let x, y = Scanf.scanf "%f %f" (fun a b -> a, b);;

if (x = 0.0 && y = 0.0) then (
    Printf.printf "Origem\n";
) else if (x = 0.0) then (
    Printf.printf "Eixo Y\n";
) else if (y = 0.0) then (
    Printf.printf "Eixo X\n";
) else if (y > 0.0 && x > 0.0) then (
    Printf.printf "Q1\n";
) else if (y > 0.0 && x < 0.0) then (
    Printf.printf "Q2\n";
) else if (y < 0.0 && x < 0.0) then (
    Printf.printf "Q3\n";
) else if (y < 0.0 && x > 0.0) then (
    Printf.printf "Q4\n";
)